### PR TITLE
maintenance: yarn format on repo

### PR DESCRIPTION
**Pull request by jsundai**
## What does this PR do?
the number of files where yarn format hasn't been run is getting increasingly large 🫣 
doing this maintenance task. 

## Notes to reviewers
need to check code samples that were auto-edited before review.

### CODE REVIEW

**Review by fairlydurable — APPROVED** [review]
Amazing job, Jwahir. Thank you for doing this.

File: docs/develop/dotnet/message-passing.mdx [hunk 1]
```diff
@@ -313,31 +314,31 @@ Also note that you can't send Updates to other Workflow Executions or perform an
...
+```csharp
+// Wait until the update is accepted
+var updateHandle = await workflowHandle.StartUpdateAsync(
+    wf => wf.SetGreetingAsync(new HelloWorldInput("World")));
+// Wait until the update is completed
+var updateResult = await updateHandle.GetResultAsync();
+```
 
-  For more details, see the "Async handlers" section.
+For more details, see the "Async handlers" section.
```
Comment (jsundai) [inline docs/develop/dotnet/message-passing.mdx:341]: This looks like this needs a link.

File: docs/develop/dotnet/message-passing.mdx [hunk 2]
```diff
@@ -595,7 +593,7 @@ When sending a Signal, Update, or Query to a Workflow, your Client might encount
   You'll receive a [`Temporalio.Exceptions.RpcException`](https://dotnet.temporal.io/api/Temporalio.Exceptions.RpcException.html) exception whose `Code` property is [`RpcException.StatusCode`](https://dotnet.temporal.io/api/Temporalio.Exceptions.RpcException.StatusCode.html) with a status of `Unavailable` (after some retries).
 
 - **The Workflow does not exist**:
-     You'll receive a [`Temporalio.Exceptions.RpcException`](https://dotnet.temporal.io/api/Temporalio.Exceptions.RpcException.html) exception whose `Code` property is [`RpcException.StatusCode`](https://dotnet.temporal.io/api/Temporalio.Exceptions.RpcException.StatusCode.html) with a status of `NotFound`.
+  You'll receive a [`Temporalio.Exceptions.RpcException`](https://dotnet.temporal.io/api/Temporalio.Exceptions.RpcException.html) exception whose `Code` property is [`RpcException.StatusCode`](https://dotnet.temporal.io/api/Temporalio.Exceptions.RpcException.StatusCode.html) with a status of `NotFound`.
 
 See [Exceptions in message handlers](/encyclopedia/workflow-message-passing#exceptions) for a non–.NET-specific discussion of this topic.
```
Comment (jsundai) [inline docs/develop/dotnet/message-passing.mdx:598]: Could probably be rephrased. "non–.NET-specific"

File: docs/develop/dotnet/message-passing.mdx [hunk 3]
```diff
@@ -614,18 +612,18 @@ When working with Updates, you may encounter these errors:
 - **No Workflow Workers are polling the Task Queue**:
   Your request will be retried by the SDK Client indefinitely.
   Use a `CancellationToken` in your [RPC options](https://dotnet.temporal.io/api/Temporalio.Client.WorkflowUpdateOptions.html#Temporalio_Client_WorkflowUpdateOptions_Rpc) to cancel the Update.
-   This raises a [Temporalio.Exceptions.WorkflowUpdateRpcTimeoutOrCanceledException](https://dotnet.temporal.io/api/Temporalio.Exceptions.WorkflowUpdateRpcTimeoutOrCanceledException.html) exception .
+  This raises a [Temporalio.Exceptions.WorkflowUpdateRpcTimeoutOrCanceledException](https://dotnet.temporal.io/api/Temporalio.Exceptions.WorkflowUpdateRpcTimeoutOrCanceledException.html) exception .
```
Comment (jsundai) [inline docs/develop/dotnet/message-passing.mdx:615]: There's an extra period here.

**Pull request closed without merging**